src/precice/bindings/python_future/test/precice-config.xml

```diff
@@ -8,8 +8,8 @@
 
   <solver-interface dimensions="2" >
    
-    <data:vector name="Forces"  />
-    <data:vector name="Velocities"  />
+    <data:scalar name="Forces"  />
+    <data:scalar name="Velocities"  />
```

Comment:
Was this indeed intended?

Reply:
Yes. I just wanted to test the functions transferring scalar data.

But I agree, the names are confusing and there should also be a test for the functions dealing with vector data.